4-texlive/texlive/texlive_packages

```diff
@@ -1,6 +1,17 @@
-texliveonfly
-latexmk
+# In the case you have to install packages manually, you can use an index of packages like
+# http://ctan.mirrors.hoobly.com/systems/texlive/tlnet/archive/
+# Or better, check https://www.ctan.org/pkg/some-package to see in which TeX Live package it is contained.
+# Or use $ tlmgr info some-package
+
+# Base tools to compile
+texliveonfly latexmk
+
+# Collections
 collection-langeuropean
 collection-fontsrecommended
 collection-fontsrecommended
+# LuaTeX
 luaotfload
+
+# texliveonfly fail whit this
```

Comment:
I'm guessing you mean that texliveonfly won't detect these ones? How did you find this list? I don't have them here yet: https://tex.stackexchange.com/a/463842/98850 (under 'known problems')